src/dsql/StmtNodes.h

```diff
@@ -1344,6 +1344,7 @@ class SelectNode final : public TypedNode<StmtNode, StmtNode::TYPE_SELECT>
 	bool dsqlForUpdate = false;
 	bool dsqlWithLock = false;
 	bool dsqlSkipLocked = false;
+	TriState dsqlOptimize;
```

Comment:
It would look for informative some name like `dsqlOptimizeForFirst` as a bool here has no meaning.